packages/backend/src/nest/connections-manager/connections-manager.service.ts

```diff
@@ -524,8 +549,17 @@ export class ConnectionsManagerService extends EventEmitter implements OnModuleI
     await this.libp2pService.createInstance(params)
 
     // Libp2p event listeners
-    this.libp2pService.on(Libp2pEvents.PEER_CONNECTED, (payload: { peers: string[] }) => {
+    this.libp2pService.on(Libp2pEvents.PEER_CONNECTED, async (payload: { peers: string[] }) => {
       this.serverIoProvider.io.emit(SocketActionTypes.PEER_CONNECTED, payload)
+      for (const peer of payload.peers) {
+        const peerStats: NetworkStats = {
+          peerId: peer,
+          connectionTime: 0,
+          lastSeen: DateTime.utc().toSeconds(),
+        }
+
+        await this.localDbService.put(LocalDBKeys.PEERS, peerStats)
```

Comment:
Do we want to use `update` here so that we don't overwrite `PEERS`? Also, it looks like `PEERS` is an object of peerStats: https://github.com/TryQuiet/quiet/blob/abd9101f84149ae4ec1db3038fca31880334cdf3/packages/backend/src/nest/connections-manager/connections-manager.service.ts#L582-L584